src/Microsoft.OData.ModelBuilder/Vocabularies/Capabilities/Constants.cs

```diff
@@ -0,0 +1,212 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+namespace Microsoft.OData.ModelBuilder.Vocabularies.Capabilities
+{
```

Comment:
I don't have any strong feelings for or against your approach here but did you consider if this structure is easier to use/work with than one where the constants are split into multiple files: `CountRestrictionsConstants.cs`, `NavigationRestrictionsConstants.cs`, etc?